packages/lit-virtualizer/src/Virtualizer.ts

```diff
@@ -606,8 +605,8 @@ export class Virtualizer {
     // a certain size, so we clamp it here (this value based on ad hoc
     // testing in Chrome / Safari / Firefox Mac)
     const max = 8200000;
-    const h = size && (size as HorizontalScrollSize).width ? Math.min(max, (size as HorizontalScrollSize).width) : 0;
-    const v = size && (size as VerticalScrollSize).height ? Math.min(max, (size as VerticalScrollSize).height) : 0;
+    const h = size && size.width !== null ? Math.min(max, size.width) : 0;
```

Comment:
Can `size.width` ever be `undefined`? In which case may want to use a less strict equality: `!=`

[Style guide reference](https://google.github.io/styleguide/tsguide.html#:~:text=Exception%3A%20Comparisons%20to%20the%20literal%20null%20value%20may%20use%20the%20%3D%3D%20and%20!%3D%20operators%20to%20cover%20both%20null%20and%20undefined%20values.)

Reply:
https://github.com/PolymerLabs/uni-virtualizer/blob/ec9ca60f8650d3fa40ea01affe9e9a0b73178144/packages/lit-virtualizer/src/Virtualizer.ts#L63-L66

Per the interface defined here, `size.width` should always be either a number or `null`.